### PR TITLE
Thread docs

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @frewscvx

### CODE REVIEW

File: src/libstd/thread/mod.rs [hunk 1]
```diff
@@ -88,43 +86,43 @@
...
 //! ## The `Thread` type
 //!
-//! Threads are represented via the `Thread` type, which you can get in one of
+//! Threads are represented via the [`Thread`] type, which you can get in one of
 //! two ways:
 //!
-//! * By spawning a new thread, e.g. using the `thread::spawn` function, and
-//!   calling `thread()` on the `JoinHandle`.
-//! * By requesting the current thread, using the `thread::current` function.
+//! * By spawning a new thread, e.g. using the [`thread::spawn`][`spawn]
```
Comment (frewsxcv) [inline src/libstd/thread/mod.rs]:
```
[`thread::spawn`][`spawn]
```

This looks like a typo.

File: src/libstd/thread/mod.rs [hunk 2]
```diff
@@ -88,43 +86,43 @@
...
+//!   function, and calling [`thread()`] on the [`JoinHandle`].
```
Comment (frewsxcv) [inline src/libstd/thread/mod.rs]: Nit: As per the docs meeting this week, we don't need the `()` in the function/method display names for links, but we can address all these later in one PR.

**Review by frewsxcv — APPROVED** [review]
r=me after the link fix

File: src/libstd/thread/mod.rs (hunk 2 above)
Comment (GuillaumeGomez) [inline src/libstd/thread/mod.rs]: I'd prefer so if you don't mind.

**Pull request merged**